app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -19,9 +20,20 @@ interface DocumentCaptureWarningProps {
 
 const DISPLAY_ATTEMPTS = 3;
 
+function getHeading({ isFailedDocType, isFailedSelfieLivenessOrQuality, t }) {
```

Comment:
Can we add type annotations for all function arguments added in this pull request?

Reply:
Hm, I see what you mean. Maybe we can reference the type from `@18f/identity-i18n` ?

e.g.

```ts
import type { I18n } from '@18f/identity-i18n';

type GetHeadingArguments = {
  // ...
  t: typeof I18n.prototype.t;
};
```

Reply:
Oh, cool! That is certainly much clearer than what I had. Thanks!

Reply:
[Added](https://github.com/18F/identity-idp/pull/9975/commits/95dbb66ab971ff14f0e8555bd42b9032d6fc45dd)